plugins/pageview/class.tx_dlf_pageview.php

```diff
@@ -55,6 +55,11 @@ class tx_dlf_pageview extends tx_dlf_plugin {
     protected function addViewerJS() {
 
         $output = array ();
+        $tmp = $this->getFulltext($this->piVars['page']);
+        $bTeiXml = false;
+        if (isset($tmp['mimetype']) && $tmp['mimetype'] == 'application/tei+xml') {
+            $bTeiXml = true;
+        }
```

Comment:
You introduced the member variable `mimetypeFulltext` in `tx_dlf_document`. Why don't you use it here?

I suggest adding a getter method for `tx_dlf_document->mimetypeFulltext` and then checking for TEI simply by comparing `$this->doc->mimetypeFulltext` to `application/tei+xml`.